react/components/Footer/global.css

```diff
@@ -93,7 +93,7 @@
   }
 
   .vtex-footer__social-networks-container {
-    padding-top: 1.5rem;
+    padding-top: 0.3rem;
```

Comment:
Can it be replaced by ```pt2```?

Reply:
It wouldn't set the same padding :/

Reply:
Could be .pa1 { padding: var(--spacing-extra-small); } --spacing-extra-small: .25rem;